src/index.js

```diff
@@ -209,12 +210,14 @@ function getConfigGetter(options) {
  * @param {string[]} directories
  * @param {string} filename
  * @param {Object} argv
+ * @param {Object} [configFromCosmiconfig]
  * @return {Object}
  */
-function getConfig(directories, filename, argv) {
+function getConfig(directories, filename, argv, configFromCosmiconfig) {
 	return Object.assign(
 		{},
 		getConfigFromFile(directories, filename),
+		configFromCosmiconfig || {},
```

Comment:
Shouldn't we call `getConfigFromCosmiconfig` here the same way we do the other two options?

Reply:
I wanted call in same way, but config required early in `mrm.js` for custom dir, preset name. 

One call for read config — potentially better performance.

Use only cosmiconfig for read config files — sound good, i try to refactor code.

Reply:
For the preset name, I guess you can swap the blocks `const options...` and `const preset...`.